domain/domain.go

```diff
@@ -824,11 +826,11 @@ func (do *Domain) Init(ddlLease time.Duration, sysFactory func(*Domain) (pools.R
 		// Local store needs to get the change information for every DDL state in each session.
 		go do.loadSchemaInLoop(ctx, ddlLease)
 	}
-	do.wg.Add(3)
+	do.wg.Add(4)
 	go do.topNSlowQueryLoop()
 	go do.infoSyncerKeeper()
+	go do.renewLease()
```

Comment:
now you have used the ```var wg util.WaitGroupWrapper``` and you can simplify the code like here.

https://github.com/pingcap/tidb/blob/b266d7d57424812404b27bf35c0b27c75f235e99/br/pkg/lightning/restore/checksum_test.go#L100

Reply:
you can only change the ```renewLease``` for using the WaitGroupWrapper.

Reply:
the other leave it as it is because ```WaitGroupWrapper``` is compatible with ```WaitGroup```.